test/test_view.py

```diff
@@ -495,11 +496,10 @@ def setUp(self):
         self.app_admin = ApplicationAdmin()
         self.person_admin = self.app_admin.get_related_admin(Person)
         self.admin_route = self.person_admin.get_admin_route()
-        self.person_model = CollectionProxy(self.admin_route)
+        columns = [ fn for fn, fa in self.person_admin.get_fields() ]
+        self.person_model = get_qml_root_backend().createModel(self.admin_route, columns)
         self.person_model.set_value(self.model_context_name)
-        list(self.person_model.add_columns(
-            [fn for fn,fa in self.person_admin.get_fields()]
-        ))
+        list(self.person_model.add_columns(columns))
```

Comment:
waarom w de columns bij constructie meegegeven, als ze ook ge-add w ?

Reply:
SetColumns verwacht momenteel dat de ColumnBackends reeds bestaan. Op zich zouden we deze ook kunnen aanmaken indien deze nog niet bestaan.